### DISCUSSION TITLE
Add landuse = retail in the impervious table

### DISCUSSION THREAD

**Discussion by j3r3m1** [orbisgis/geoclimate#833] [Ideas]
It seems that the retail zones in OSM are not identified as impervious. We might add that in the impervious layer (https://github.com/orbisgis/geoclimate/blob/247dd2f2ee0d5789a901ce6d9ce7862618180049/osm/src/main/resources/org/orbisgis/geoclimate/osm/imperviousParams.json#L20) ? It would allow to classify the following zones as impervious:
- https://www.openstreetmap.org/way/27218166#map=16/43.6101/1.3084
- https://www.openstreetmap.org/way/123066264#map=15/43.5331/1.4045
- https://www.openstreetmap.org/way/238307044 (sat: https://graphhopper.com/maps/?point=47.651068%2C-2.084214_Redon%2C+35600%2C+Bretagne%2C+France&point=&profile=car&layer=Esri+Satellite)
- https://www.openstreetmap.org/way/249743977#map=15/45.5965/5.8986 (sat: https://graphhopper.com/maps/?point=45.596524%2C5.898415_Les+Landiers%2C+73000+Chamb%C3%A9ry%2C+Auvergne-Rh%C3%B4ne-Alpes%2C+France&point=&profile=car&layer=Esri+Satellite)

**Comment by j3r3m1** [comment]
OK we can add it in this impervious json param

**Comment by ebocher** [comment]
Done
